basetest.pm

```diff
@@ -320,11 +320,12 @@ sub post_run_hook {
 sub run_post_fail {
     my ($self, $msg) = @_;
     return if $bmwqemu::vars{_SKIP_POST_FAIL_HOOKS};
+    $self->{result}                 = 'fail' if $self->{result};
```

Comment:
you added the result 'fail' here. Maybe you can use 'softfail'.

See https://github.com/os-autoinst/os-autoinst/blob/2bd444f4840a223602021e564d092fd6dc4ac69b/basetest.pm#L460

Reply:
I don't want to change the existing behaviour.
I just want to allow overwriting the result from post fail hook.